### PR TITLE
[stable/yugaware] remove 'release' key from configMaps

### LINKED ISSUE #1398: [YW][k8s] helm chart validation error on configmaps
Using helm 2.14.0; tried to install yw-1.0.0 chart, and got the following validation error:

```
Release "yugaware" does not exist. Installing it now.
Error: validation failed: error validating "": error validating data: [ValidationError(StatefulSet.spec.template.spec.volumes[4].configMap): unknown field "release" in io.k8s.api.core.v1.ConfigMapVolumeSource, ValidationError(StatefulSet.spec.template.spec.volumes[5].configMap): unknown field "release" in io.k8s.api.core.v1.ConfigMapVolumeSource]
```

If we take a look in `yugaware/templates/service.yml` we see this:

```yml
spec:
  serviceName: {{ .Release.Name }}-yugaware
  replicas: {{ .Values.yugaware.replicas }}
  selector:
    matchLabels:
      app: {{ .Release.Name }}-yugaware
  template:
    metadata:
      labels:
        app: {{ .Release.Name }}-yugaware
    spec:
      serviceAccountName: {{ .Release.Name }}
      imagePullSecrets:
      - name: {{ .Values.image.pullSecret }}
      volumes:
        - name: yugaware-storage
          persistentVolumeClaim:
            claimName: {{ .Release.Name }}-yugaware-storage
        - name: yugaware-ui
          emptyDir: {}
        - name: thirdparty-deps
          emptyDir: {}
        - name: prometheus-storage
          emptyDir: {}
        - name: yugaware-config
          configMap:
            name: {{ .Release.Name }}-yugaware-app-config
            release: {{ .Release.Name }}
            items:
              - key: application.docker.conf
                path: application.docker.conf
```

notice the `configMap`'s have a `release:` key, which [I don't think has `release` as a valid subkey under it?](https://kubernetes.io/docs/reference/federation/extensions/v1beta1/definitions/#_v1_configmapvolumesource)

I'm not sure if this is an issue which is a bug in [helm-2.14.0, possibly because of this issue](https://github.com/helm/helm/commit/32d7f1a3fc226c1745a58b51e318b7362bc7a0bf), or if it's a minor hiccup in the chart & it wasn't caught until something happened with manifest validation in 2.14.0...?

Thanks for your time

### PR DESCRIPTION
**Pull request by aegershman**
closes https://github.com/YugaByte/yugabyte-db/issues/1398

recreates PR from: https://github.com/YugaByte/charts/pull/3

### CODE REVIEW

File: stable/yugaware/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: 1.3.2.0-b18
-version: 1.3.2
+version: 1.3.3
```
Comment (ramkumarvs) [inline stable/yugaware/Chart.yaml:3]: @aegershman hey chatted internally with the team, and we all agree we need to bump the chart version as you pointed out, but given we want to keep our version (3 digits) to be inline with our stable release version. I am proposing any changes before that we will have committers tag them prerelease-X. so for this revision the version would be 1.3.2-prerelease-2

**Review by ramkumarvs — APPROVED** [review]

File: stable/yugaware/Chart.yaml (hunk 1 above)
Comment (aegershman) [reply stable/yugaware/Chart.yaml:3]: so should this be a bump to the next version? as in `1.3.3-prerelease-2`?

Comment (ramkumarvs) [reply stable/yugaware/Chart.yaml:3]: yup

**Review by ramkumarvs — COMMENTED** [review]
dang forgot to submit the review.

**Comment by aegershman** [comment]
Hi from the future! Closing to prune dead PRs. Hope all is well 🫡

**Pull request closed without merging**